modules/developer-cli-odo-switching-between-ephemeral-and-persistent-storage.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// *cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/working-with-storage.adoc
+
+[id="switching-between-ephemeral-and-persistent-storage_{context}"]
+= Switching between ephemeral and persistent storage
+
+You can switch between ephemeral and persistent storage in your project by using the `odo preference` command. `odo preference` modifies the global preference in your cluster.
+
+When persistent storage is enabled, the cluster stores the information between the restarts. 
+
+When ephemeral storage is enabled, the cluster stores temporary information that does not need to persist between the restarts.
+
+Persistent storage is enabled by default. 
+
+.Procedure
+
+. To set the ephemeral storage:
++
+[source,terminal]
+----
+$ odo preference set ephemeral true
+----
+
+. Push the changes to the cluster:
```

Comment:
We don't necessarily need `odo push` to apply Ephemeral vs. Persistent thing. However, it is true that only `odo push` would help the user see the change take place on their component.

Thing is, this being a Preference setting, it applies to all components currently deployed by the user as well as ones they would deploy in future.

I'm more inclined towards using a statement like below:
> Now doing `odo push` for any of your component(s) would result in odo creating an Ephemeral storage for it.

Finally, since this section of doc (`To set the ephemeral storage:`) is written to help user switch from Persistent to Ephemeral, can we modify it to help them convert from Ephemeral to Persistent instead? Asking for this becasue Ephemeral is default.

Reply:
Thank you, @dharmit 
>Thing is, this being a Preference setting, it applies to all components currently deployed by the user as well as ones they would deploy in future.
It's project-specific, or am I wrong? Each project has its own config.

I've made some changes - added commands on how to bot set and unset, PTAL when you can. 

Although now I'm thinking - maybe it's better to document:
>odo preference view
>odo preference unset ephemeral
>odo push
>oc get po,pvc 
>odo preference set ephemeral
>odo push
>oc get po,pvc 

It's a lot of steps but it will show how to set and check the config and actually see whether the persistent storage has been created.